squad/core/templates/squad/notification/diff.txt.jinja2

```diff
@@ -34,9 +34,10 @@ Failures
 Threshold(s) exceeded
 ---------------------
 {% for threshold_metric in thresholds %}
-Metric name: {{threshold_metric[1].name}}
-Metric result: {{threshold_metric[1].result}}
-Threshold value: {{threshold_metric[0].value}}
+Environment: {{threshold_metric[0].slug}}
```

Comment:
Continuing my comment above, here instead of having `threshold_metric[0].slug` you can have `threshold_metric[2].environment.slug`, this is, removing `environment` from the tuple, the indexes would change accordingly.